index.md

```diff
@@ -62,6 +62,14 @@ commands, such as `SHOW TABLES` or `SELECT/INSERT/UPDATE/DELETE`.
 
 * [GitHub repository](https://github.com/cloudspannerecosystem/spanner-cli)
 
+### wrench
+
+* [GitHub repository](https://github.com/cloudspannerecosystem/wrench)
```

Comment:
For consistency with the other sections, please move the link after the description.